notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -83,7 +83,9 @@
     "import hera_pspec as hp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        avg_data_copy = copy.deepcopy(avg_data)
</pre><p>This feels kinda hacky. I think we can probably architect the main analysis cell (which does the FRF, coherent time avg, and the psuedo stokes) better so that these keys aren't here.</p><p> </p><p>Or change form pseudo-stokes to create a brand-new DataContainer rather than adding keys to an existing one. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>